Sources/KSCrashRecordingCore/KSDate.c

```diff
@@ -25,32 +25,48 @@
 #include "KSDate.h"
 
 #include <stdio.h>
+#include <string.h>
 #include <sys/time.h>
 #include <time.h>
 
-void ksdate_utcStringFromTimestamp(time_t timestamp, char *buffer21Chars)
+void ksdate_utcStringFromTimestamp(time_t timestamp, char *buffer, size_t bufferSize)
 {
+    memset(buffer, 0, bufferSize);
```

Comment:
Technically, only [0] needs to be zero since `snprintf` guarantees null termination no matter what, but this isn't a critical path so no worries.